README.yaml

```diff
@@ -73,7 +72,7 @@ usage: |-
 
   ```hcl
   module "example" {
-    source = "https://github.com/cloudposse/terraform-example-module.git?ref=master"
+    source  = "https://github.com/cloudposse/terraform-helm-release.git?ref=master"
     example = "Hello world!"
   }
   ```
```

Comment:
Please add a relevant snippet and make sure `source` points to a release in the Terraform registry (and not a git ref).

Reply:
doesn't it have to be in the registry first?

Reply:
@nitrocode not necessarily. `release-drafter` requires you to draft an initial release first, which should probably be `0.1.0`. You can just create a snippet to show how to use this module and set the version to `0.1.0`.

Reply:
Would the source be this?

```
  source  = "cloudposse/release/helm"
  version = "0.1.0"
```

Reply:
Done